docs/pages/contrib/hypothesis_plugins.rst

```diff
@@ -140,8 +108,26 @@ like ``Future``, ``ReaderFutureResult``, etc
 that have complex ``__init__`` signatures.
 And we don't want to mess with them.
 
-You can also register a custom strategy to be used when running your
-container's laws:
+Warning::
+  Checking laws is not compatible with ``pytest-xdist``,
+  because we use a lot of global mutable state there.
+  Please, use ``returns_lawful`` marker
+  to exclude them from ``pytest-xdist`` execution plan.
```

Comment:
We can also make a note about running them on a single node.

Reply:
Ah, this was an existing paragraph that I moved up. I don't have anything to add here.